CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # MicroProfile Language Server Changelog
 
+## [0.12.0](https://github.com/eclipse/lsp4mp/milestone/13?closed=1) (August 19, 2024)
```

Comment:
We need to punt the release to Monday, August 26th, since Red Hat is closed on Friday, we don't want to take any chance if something goes south.

cc @aparnamichael @MituuZ @turkeylurkey @mrglavas